lib/typescript.js

```diff
@@ -44,13 +44,13 @@ module.exports = {
       {
         selector: ["variable"],
         modifiers: ["const"],
-        format: ["camelCase", "UPPER_CASE", "snake_case"],
+        format: ["camelCase", "UPPER_CASE", "snake_case", "PascalCase"],
```

Comment:
This is not how you should be doing react changes - the way that it should be is that we have another file in the `config` for `react` that would then pull in different rules from the `lib` folder.

Reply:
Okay, can I have removed those changes for now and will create a separate PR with a bigger scope for React. 

This is currently how I've been using it in a React project but I imagine we can extend it within the library rather than here. I'd like to base it off `react-app` and layer our other rules. 

```json
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest",
      "@elementx-ai/eslint-config/configs/typescript"
    ],
    "rules": {
      "@typescript-eslint/naming-convention": "off"
    }
  },
```